tests/api_fastapi/core_api/routes/public/test_dag_run.py

```diff
@@ -146,14 +148,41 @@ def setup(request, dag_maker, session=None):
     )
 
     dag_maker.sync_dagbag_to_db()
-    dag_maker.dag_model
     dag_maker.dag_model.has_task_concurrency_limits = True
     session.merge(ti1)
     session.merge(ti2)
     session.merge(dag_maker.dag_model)
     session.commit()
 
 
+def get_dag_versions_dict(dag_version: DagVersion):
+    return DagVersionResponse.model_validate(dag_version, from_attributes=True).model_dump(mode="json")
```

Comment:
I think we can omit it
```suggestion
    return DagVersionResponse.model_validate(dag_version, from_attributes=True).model_dump()
```

Reply:
We must set `mode="json"` (the default is `mode="python"`) or `created_at` and `id` will be Python `datetime` and `UUID` objects instead of strings.